client/packages/purchasing/src/purchase_order/DetailView/LineEdit/PurchaseOrderLineEdit.tsx

```diff
@@ -164,47 +223,6 @@ export const PurchaseOrderLineEdit = ({
                     decimalLimit: 2,
                   }
                 )}
-              {textInput('label.unit', draft?.unit || '', value =>
```

Comment:
Did some shufflin' around so that we'd have space for other rows in the left column